stream_chat/async_chat/channel.py

```diff
@@ -35,9 +35,12 @@ async def delete_reaction(
             params={"user_id": user_id},
         )
 
-    async def create(self, user_id: str) -> StreamResponse:
```

Comment:
you'd need to make the same change to the sync client as well https://github.com/GetStream/stream-chat-python/blob/master/stream_chat/channel.py#L33
this split between sync/async exists in the tests too

additionally, you'd want to update the signature in the base class too https://github.com/GetStream/stream-chat-python/blob/master/stream_chat/base/client.py

Reply:
Thanks Guyon! Just updated it.

Reply:
I think it's still missing a test in `stream_chat/tests/async_chat
/test_channel.py`